pkg/api/dashboard.go

```diff
@@ -238,6 +238,11 @@ func (hs *HTTPServer) GetDashboard(c *contextmodel.ReqContext) response.Response
 		Meta:      meta,
 	}
 
+	// do not expose internal ids... stuff will likely break!!!!!
+	if false {
+		dash.Data.Del("id") // do not advertise the internal ids
```

Comment:
ideally we never tell the frontend about the internal primary key... but there are still things that use it 😢